assets/js/state/clusters.test.js

```diff
@@ -0,0 +1,18 @@
+import clustersReducer, { removeCluster } from '@state/clusters';
+
+describe('Clusters reducer', () => {
+  it('should remove cluster from state', () => {
+    const cluster = { id: 'test-cluster-id' };
```

Comment:
Most probably it would be better to use the `clusterFactory`.
```
const { id } = clusterFactory.build()
# or
const clusters = cluster.Factory.buildList(5) # with some further work, which would do a more robust test, just removing 1 from the list, instead of looking for an empty list
```

The same goes for the other tests